File: Python Fundamentals/Functions - Exercise/odd_and_even_sum.py
def summed(number):
    even_digits = 0
    odd_digits = 0
    string_to_int = []
    for string in number:
        string_to_int.append(int(string))
    for num in string_to_int:
        if num % 2 == 0:
            even_digits += num
        elif num % 2 != 0:
            odd_digits += num
    return f"Odd sum = {odd_digits}, Even sum = {even_digits}"


number = input()
print(summed(number))

File: JS Front-End/Functions and Statements - Exercises/oddAndEvenSum.js
function oddAndEvenSum(n) {
  let evenSum = 0;
  let oddSum = 0;

  let nAsString = n.toString();

  for (let i = 0; i < nAsString.length; i++) {
    Number(nAsString[i]) % 2 == 0
      ? (evenSum += Number(nAsString[i]))
      : (oddSum += Number(nAsString[i]));
  }

  console.log(`Odd sum = ${oddSum}, Even sum = ${evenSum}`);
}

oddAndEvenSum(1000435);
